apps/server/src/modules/files-storage/controller/api-test/files-storage-download-upload.api.spec.ts

```diff
@@ -317,8 +302,8 @@ describe('files-storage controller (API)', () => {
 
 			describe(`with already existing file`, () => {
 				beforeEach(async () => {
-					const expectedResponse1 = createFileResponse();
-					const expectedResponse2 = createFileResponse();
+					const expectedResponse1 = TestHelper.createFile('bytes 0-3/4');
+					const expectedResponse2 = TestHelper.createFile('bytes 0-3/4');
```

Comment:
```suggestion
					const expectedResponse1 = TestHelper.createFile();
					const expectedResponse2 = TestHelper.createFile();
```

Originally the created file response did use no bytes range.